cypress/e2e/cloud/home.test.ts

```diff
@@ -1,63 +0,0 @@
-import {Organization} from '../../../src/types'
```

Comment:
I don't know anything about the `alertsActivity` flag, but it does still appear to exist (it's set to `false`) in prod. Do we need to move these tests over to new firstMile page, rather than removing them?

Reply:
No, since the `AlertsActivity` component isn't part of first mile right now

Reply:
Great! 👍